Yoast/wordpress-seo
# Add Structured Data How To block for use in Gutenberg.

## Summary

This PR can be summarized in the following changelog entry:

* Adds a How To block for use in Gutenberg to correctly format structured data for how-to guides according to schema.org.

## Relevant technical choices:

* Uses Gutenberg globals from the window object since these currently aren't defined as externals.

## Test instructions

This PR can be tested by following these steps:

* Create a new post.
* Add a how-to block.
* Create a guide.
* Try moving steps around, pasting content and other things.
* Save the post.
* Check if the output contains all your data and has a valid JSON-LD blob above it ( you can use https://search.google.com/structured-data/testing-tool ).

## Quality assurance


Requires #10373
